### ISSUE TITLE
Pelican 4.0.0 breaks plugin `i18n_subsites` again

### ISSUE THREAD

**Issue by siongui** [getpelican/pelican-plugins#1086]
After upgrading to Pelican 4.0.0, plugin `i18n_subsites` does not work

````
ERROR: Cannot load plugin `i18n_subsites`
  | ImportError: cannot import name Draft
CRITICAL: UndefinedError: 'relpath_to_site' is undefined
````
[detail error log on Travis CI](https://travis-ci.org/siongui/ajahn-jayasaaro/builds/457595606)

I googling the error message and found this happened on #937 before. Looks like this is the same issue?

**Comment by avaris** [comment]
Will be fixed with #1084

**Comment by justinmayer** [comment]
Fix has been merged via above PR.
